docs/install.rst

```diff
@@ -247,138 +228,41 @@ The following packages can optionally be used when testing:
-to build ``astropy``. You do not need to install any other specific build
-dependencies (such as `Cython <https://cython.org/>`_) since these are
-declared in the ``pyproject.toml`` file and will be automatically installed into
-a temporary build environment by pip.
-
-Prerequisites for Linux
-=======================
-
-On Linux, using the package manager for your distribution will usually be the
-easiest route to making sure you have the prerequisites to build ``astropy``. In
-order to build from source, you will need the Python development
-package for your Linux distribution, as well as pip.
-
-For Debian/Ubuntu::
-
-    sudo apt-get install python3-dev python3-numpy-dev python3-setuptools cython3 python3-pytest-astropy
-
-For Fedora/RHEL::
-
-    sudo yum install python3-devel python3-numpy python3-setuptools python3-Cython python3-pytest-astropy
-
-.. note:: Building the developer version of ``astropy`` may require
-          newer versions of the above packages than are available in
-          your distribution's repository.  If so, you could either try
-          a more up-to-date distribution (such as Debian ``testing``),
-          or install more up-to-date versions of the packages using
-          ``pip`` or ``conda`` in a virtual environment.
-
-Prerequisites for Mac OS X
-==========================
-
-On MacOS X you will need the XCode command line tools which can be installed
-using::
-
-    xcode-select --install
-
-Follow the onscreen instructions to install the command line tools required.
-Note that you do **not** need to install the full XCode distribution (assuming
-you are using MacOS X 10.9 or later).
-
-The `instructions for building NumPy from source
-<https://numpy.org/doc/stable/building/index.html>`_ are a good
-resource for setting up your environment to build Python packages.
-
-Obtaining the Source Packages
-=============================
-
-Source Packages
----------------
-
-The latest stable source package for ``astropy`` can be `downloaded here
-<https://pypi.org/project/astropy>`_.
-
-Development Repository
-----------------------
-
-The latest development version of ``astropy`` can be cloned from GitHub
-using this command::
+If you want to build the code from source, follow the instructions for
+:ref:`contributing_environment`. Note that instead of cloning from your fork, you can
+choose to clone from the main repository::
 
-   git clone https://github.com/astropy/astropy.git
+    git clone https://github.com/astropy/astropy.git
+    cd astropy
 
-If you wish to participate in the development of ``astropy``, see the
-:ref:`developer-docs`. The present document covers only the basics necessary to
-installing ``astropy``.
+Building the documentation is typically not necessary unless you are
+developing code or documentation or do not have internet access, because
+the latest (and archive) versions of Astropy's documentation are
+available at `docs.astropy.org <https://docs.astropy.org>`_ . The process
```

Comment:
This needs clarification because https://docs.astropy.org routes to stable doc so it is very confusing.

Reply:
Updated to:
```
... because the stable, latest, and archive versions of Astropy's documentation are
available at `docs.astropy.org <https://docs.astropy.org>`_ .
```